gmm_client/test/api_coverage_SUITE.erl

```diff
@@ -0,0 +1,211 @@
+
+    % then
+    [
+        ?assertEqual("true", client_test_helper:check_edge_existance("zone1/User1", "zone1/Space1")),
+        ?assertEqual("\"01110\"", client_test_helper:check_permissions("zone1/User1", "zone1/Space1")),
+
+        ?assertEqual("true", client_test_helper:check_edge_existance("zone1/User1", "zone1/Group2")),
+        ?assertEqual("\"10101\"", client_test_helper:check_permissions("zone1/User1", "zone1/Group2")),
+
+        ?assertEqual("true", client_test_helper:check_edge_existance("zone1/Group2", "zone1/Provider1")),
+        ?assertEqual("\"10001\"", client_test_helper:check_permissions("zone1/Group2", "zone1/Provider1")),
+
+        ?assertEqual(0, length(User1Parents)),
+        ?assertEqual(2, length(User1Children)),
+
+        ?assertEqual(1, length(Space1Parents)),
+        ?assertEqual(0, length(Space1Children)),
+
+        ?assertEqual(1, length(Group2Parents)),
+        ?assertEqual(1, length(Group2Children)),
+
+        ?assertEqual(1, length(Provider1Parents)),
+        ?assertEqual(0, length(Provider1Children)),
+
+        ?assert(lists:member("\"zone1/Group2\"", User1Children)),
+        ?assert(lists:member("\"zone1/Space1\"", User1Children)),
+        ?assert(lists:member("\"zone1/Provider1\"", Group2Children)),
+
+        ?assert(lists:member("\"zone1/User1\"", Space1Parents)),
+        ?assert(lists:member("\"zone1/User1\"", Group2Parents)),
+        ?assert(lists:member("\"zone1/Group2\"", Provider1Parents))
+    ].
+
+edges_updating_test(_Config) ->
+    % when
+    client:set_edge_permissions("zone1/User1", "zone1/Group2", "00010", "trace2", true),
+    client:set_edge_permissions("zone1/Group2", "zone1/Provider1", "11111", "trace3", false),
+
+    % then
+    [
+        ?assertEqual("\"01110\"", client_test_helper:check_permissions("zone1/User1", "zone1/Space1")),
+        ?assertEqual("\"00010\"", client_test_helper:check_permissions("zone1/User1", "zone1/Group2")),
+        ?assertEqual("\"11111\"", client_test_helper:check_permissions("zone1/Group2", "zone1/Provider1"))
+    ].
+
+edges_deleting_test(_Config) ->
+    % when
+    client:delete_edge("zone1/User1", "zone1/Space1", "trace1", false),
+    client:delete_edge("zone1/Group2", "zone1/Provider1", "trace3", false),
+
+    User1Parents = client_test_helper:parents_list("zone1/User1"),
+    User1Children = client_test_helper:children_list("zone1/User1"),
+
+    Space1Parents = client_test_helper:parents_list("zone1/Space1"),
+    Space1Children = client_test_helper:children_list("zone1/Space1"),
+
+    Group2Parents = client_test_helper:parents_list("zone1/Group2"),
+    Group2Children = client_test_helper:children_list("zone1/Group2"),
+
+    Provider1Parents = client_test_helper:parents_list("zone1/Provider1"),
+    Provider1Children = client_test_helper:children_list("zone1/Provider1"),
+
+    % then
+    [
+        ?assertEqual("false", client_test_helper:check_edge_existance("zone1/User1", "zone1/Space1")),
```

Comment:
Ta funkcja mogłaby zwracać boolean() zamiast tekstu, ale to NA RAZIE nie jest priorytet